docs/contribute/cumulative-docs/example-scenarios.md

```diff
@@ -0,0 +1,545 @@
+Just the syntax slightly differs so that it stays inline: `` {applies_to}`ech: ga` {applies_to}`serverless: ga` ``.
+:::
+
+### If labeling serverless and stateful [not-one-section]
+
+For example, if a whole page is generally applicable to Elastic Stack 9.0.0 and to Serverless, but one specific section isn’t applicable to Serverless. The content on the [Spaces](https://www.elastic.co/docs/deploy-manage/manage-spaces) page is generally applicable to both Serverless and stateful, but one section only applies stateful:
+
+* In the frontmatter, specify that the content on the page applies to both unless otherwise specified.
+
+% FOR THE REVIEWER: WHICH IS CORRECT?
+% The existing docs and the "real" example in the wild use different strategies.
+% Do we include `stack: ga` in the section annotation or not?
+* In a section-level annotation, specify that the content is `ga` in `stack` and is `unavailable` in `serverless`.
+
+:::::{tab-set}
+::::{tab-item} Image
+:::{image} ./images/example-not-one-section.png
+:screenshot:
+:alt:
+:::
+::::
+::::{tab-item} Code
+````
+---
+applies_to:
+  stack: ga
+  serverless: ga
+---
+
+# Spaces
+
+[...]
+
+## Configure a space-level landing page [space-landing-page]
+
+```{applies_to}
+stack: ga
+serverless: unavailable
+```
+````
+::::
+:::::
+
+:::{tip}
+Likewise, when the difference is specific to just one paragraph or list item, the same rules apply.
+Just the syntax slightly differs so that it stays inline: `` {applies_to}`serverless: unavailable` ``.
+:::
+
+## Functionality is added to an unversioned product [unversioned-added]
+
+When functionality is _first added_ to an unversioned product/deployment mode,
+how it is labeled depends on if the functionality is in technical preview, beta, or GA.
+
+### If the section lifecycle is the same as the page level [unversioned-added-same]
+
+For example, on the [Project settings](https://www.elastic.co/docs/deploy-manage/deploy/elastic-cloud/project-settings#obs-serverless-project-features) page we added content about the Observability Logs Essentials feature tier, that was added to Serverless in GA.
+Since the page's frontmatter already includes `serverless: ga`, there is no need to label the added content.
+
+% FOR THE REVIEWER: IS THIS TRUE?
+::::::{important}
+However, if the functionality is also applicable to a specific version of a versioned product/deployment mode,
+label the content with both versioned and unversioned applicability information.
+
+For example, on the [Lens](https://www.elastic.co/docs/explore-analyze/visualize/lens) page we added information
+about a new option that was added to Serverless in GA and the Elastic Stack in GA in 9.1.0.
+Even though it is added to Serverless, an unversioned product, in the same lifecycle state as the page-level annotation,
+we still include an inline annotation to make it clear that this is not only available in the Elastic Stack.
```

Comment:
I don't think we should do this. my biggest reason is dev contributors. they have tunnel vision for the change they're making, and serverless is often inconsequential to them. we should only need them to put the relevant stack label. 

if we need both labels to show, we should do it with code logic.